cadc-access-control/src/main/java/ca/nrc/cadc/gms/GroupClientImpl.java

```diff
@@ -0,0 +1,127 @@
+ *  written permission.                  préalable et particulière
+ *                                       par écrit.
+ *
+ *  This file is part of the             Ce fichier fait partie du projet
+ *  OpenCADC project.                    OpenCADC.
+ *
+ *  OpenCADC is free software:           OpenCADC est un logiciel libre ;
+ *  you can redistribute it and/or       vous pouvez le redistribuer ou le
+ *  modify it under the terms of         modifier suivant les termes de
+ *  the GNU Affero General Public        la “GNU Affero General Public
+ *  License as published by the          License” telle que publiée
+ *  Free Software Foundation,            par la Free Software Foundation
+ *  either version 3 of the              : soit la version 3 de cette
+ *  License, or (at your option)         licence, soit (à votre gré)
+ *  any later version.                   toute version ultérieure.
+ *
+ *  OpenCADC is distributed in the       OpenCADC est distribué
+ *  hope that it will be useful,         dans l’espoir qu’il vous
+ *  but WITHOUT ANY WARRANTY;            sera utile, mais SANS AUCUNE
+ *  without even the implied             GARANTIE : sans même la garantie
+ *  warranty of MERCHANTABILITY          implicite de COMMERCIALISABILITÉ
+ *  or FITNESS FOR A PARTICULAR          ni d’ADÉQUATION À UN OBJECTIF
+ *  PURPOSE.  See the GNU Affero         PARTICULIER. Consultez la Licence
+ *  General Public License for           Générale Publique GNU Affero
+ *  more details.                        pour plus de détails.
+ *
+ *  You should have received             Vous devriez avoir reçu une
+ *  a copy of the GNU Affero             copie de la Licence Générale
+ *  General Public License along         Publique GNU Affero avec
+ *  with OpenCADC.  If not, see          OpenCADC ; si ce n’est
+ *  <http://www.gnu.org/licenses/>.      pas le cas, consultez :
+ *                                       <http://www.gnu.org/licenses/>.
+ *
+ *  $Revision: 4 $
+ *
+ ************************************************************************
+ */
+
+package ca.nrc.cadc.gms;
+
+import java.net.URI;
+import java.security.AccessControlException;
+import java.security.cert.CertificateExpiredException;
+import java.security.cert.CertificateNotYetValidException;
+import java.util.List;
+
+import org.apache.log4j.Logger;
+
+import ca.nrc.cadc.ac.client.GMSClient;
+import ca.nrc.cadc.cred.client.CredUtil;
+
+/**
+ * This class is acting as a bridge between the GroupClient in cadc-gms
+ * and the cadc-access-control implementation named GMSClient.
+ * 
+ * In cadc-gms, GroupUtil, the group client is loaded by trying to load
+ * a class with the name ca.nrc.cadc.gms.GroupClientImpl.
+ * 
+ * This class will be removed when GMSClient is renamed to be the name
+ * of this class.
+ * 
+ * @author majorb
+ *
+ */
+public class GroupClientImpl implements GroupClient {
+    
+    Logger log = Logger.getLogger(GroupClientImpl.class);
+    
+    private GMSClient gmsClient;
```

Comment:
Why is this a wrapper around GMSClient instead of just "extends GMSClient"?  That seems to be a viable way to provide the impl class with a specific name...